include/vitals.inc.php

```diff
@@ -67,6 +67,7 @@ function unregister_GLOBALS() {
 //if (!defined('AC_REDIRECT_LOADED')){
 //	require_once(AC_INCLUDE_PATH.'lib/mysql_connect.inc.php');
 //}
+	$db =mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME, DB_PORT);
```

Comment:
Why is a db connection established here. Is the connection only needed in DAO classes?

Reply:
Will fix that now

Reply:
Thanks. Can you take the chance to remove commented lines?

Reply:
Will do that